### DISCUSSION TITLE
Trying to access role with user pagination not working

### DISCUSSION THREAD

**Discussion by umar28dec** [spatie/laravel-permission#1803] [Q&A] [answered]
Not getting roles here 

        $paginator = User::with('roles')->orderBy('created_on','desc')->paginate(10);
in this case not returning role.

If remove orderBy then it works (mean return data with roles)

        $paginator = User::with('roles')->paginate(10);

**Comment by erikn69** [comment, marked as answer]
I tried and it returns the expected result
```php
$paginator = User::with('roles')->orderBy('created_at','desc')->paginate(10);
```
```
     #relations: array:1 [▼
          "roles" => Illuminate\Database\Eloquent\Collection {#2175 ▼
            #items: array:2 [▼
              0 => App\Models\Role {#2202 ▶}
              1 => App\Models\Role {#2205 ▶}
            ]
          }
        ]
```

**Reply by umar28dec** [reply]
@erikn69 is that because of column name change **created_on**, I am using Laravel 8

**Reply by umar28dec** [reply]
@erikn69 do you want me to post complete code ?

**Reply by erikn69** [reply]
I tried changing my db field from `created_at` to `created_on` and works the same, laravel 8

**Reply by umar28dec** [reply]
Thanks for update
